src/gen.jl

```diff
@@ -149,7 +149,8 @@ function qualify_in_hierarchy(name::String, scope::Scope)
     elseif isdefined(scope, :parent)
         return qualify_in_hierarchy(name, scope.parent)
     else
-        error("unresolved name $name at scope $(scope.name)")
+        @warn "unresolved name $name at scope $(scope.name)"
```

Comment:
This is not necessary. Instead, updating the `qualify_in_hierarchy` method to something like this should achieve the intended purpose:

```
function qualify_in_hierarchy(name::String, scope::Scope)
    if startswith(name, "ProtoBuf.google.protobuf.")
        return name
    elseif name in scope.syms
        return fullname(scope, name)
    elseif isdefined(scope, :parent)
        return qualify_in_hierarchy(name, scope.parent)
    else
        error("unresolved name $name at scope $(scope.name)")
    end
end
```